docs/java/stream-apis/Using stream with custom objects.md

```diff
@@ -0,0 +1,77 @@
+---
+id: Using stream with custom objects
```

Comment:
Replace `id: Using stream with custom objects` to `id: using-stream-with-custom-objects`